bundles/statistics/statsgrid2016/components/IndicatorList.js

```diff
@@ -66,8 +66,9 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorList', function (servi
         var indicators = me._getIndicators();
         indicators.forEach(function (ind, id) {
             me.service.getUILabels(ind, function (labels) {
+                const regexFourDigitYearBetweenParenthesesAtTheEnd = /\(\d{4}\)$/;
                 var indElem = jQuery(me.__templates.indicator({
-                    name: labels.full,
+                    name: Oskari.util.cutSentenceToLength(labels.full, 60, regexFourDigitYearBetweenParenthesesAtTheEnd),
```

Comment:
You can use other keys from labels variable to reconstruct the parameters part and just "cut" the name part. This way you don't need the util method or regexp.

Reply:
Changed implementation to construct indicator name from indicator and params.